packages/shorebird_cli/lib/src/shorebird_validation_mixin.dart

```diff
@@ -4,7 +4,7 @@ import 'package:shorebird_cli/src/validators/validators.dart';
 
 mixin ShorebirdValidationMixin on ShorebirdCommand {
   /// Runs [Validator.validate] on all [validators] and writes issues to stdout.
-  Future<void> logValidationIssues() async {
+  Future<List<ValidationIssue>> logValidationIssues() async {
```

Comment:
I don't think we should keep this named `logValidationIssues` when it also returns the list.